api/graphql/schema.graphql

```diff
@@ -1943,7 +1931,7 @@ type Mutation {
   # NOT USED RIGHT NOW
   processStripeToken(postId: ID, token: String, amount: Int): GenericResult
   # react to a post or comment
-  reaction(entityId: ID, data: ReactionInput): Post
+  reactOn(entityId: ID, data: ReactionInput): Post
```

Comment:
this needs to be changed throughout no?

Reply:
Just saw that my last push failed, re-pushed